### PR TITLE
Feat: enhance requests to api

### PR DESCRIPTION
**Pull request by hiba9201**
- added qs for creating queries
- fix typo in Sources

### CODE REVIEW

File: common/data/base/houseBase.ts [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
     async getObjectsPolygonsByRange(from: number, to: number, filterName: string) {
-        const fullUrl = new URL(`${STRAPI_BASE_URL}/house?populate=borders`);
+        const url = `${STRAPI_BASE_URL}/house`;
 
-        fullUrl.searchParams.append(`filters[${filterName}][$gte]`, from.toString());
-        fullUrl.searchParams.append(`filters[${filterName}][$lte]`, to.toString());
+        const query = qs.stringify({
+            filters: {
+                [filterName]: {
+                    [FilterOperator.GreaterThanOrEqual]: from,
```
Comment (a-chabin) [inline common/data/base/houseBase.ts:17]: `FilterOperator['=>']` or `FilterOperator['≥']`

or maybe we can try to make such a thing? 😏

**Comment by a-chabin** [comment]
LGTM :)

thanks for improving DX!

**Review by a-chabin — APPROVED** [review]

File: common/data/base/houseBase.ts (hunk 1 above)
Comment (hiba9201) [reply common/data/base/houseBase.ts:17]: Then we need copy this symbol every time. i think it's uncomfortable

Comment (hiba9201) [reply common/data/base/houseBase.ts:17]: It's about second variant

**Pull request merged**